src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationAboutToStartHandler.java

```diff
@@ -58,14 +58,19 @@ public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(Finrem
 
         List<GeneralApplicationCollectionData> existingGeneralApplication =
             helper.getGeneralApplicationList(caseData, GENERAL_APPLICATION_COLLECTION);
+
+        log.info("GA AboutToStartHandler beginning existing General Application List size: {} for case ID: {}",
+            existingGeneralApplication.size(), caseId);
+
         GeneralApplicationCollectionData data =
             helper.mapExistingGeneralApplicationToData(caseData, userAuthorisation, caseId);
 
         String loggedInUserCaseRole = assignCaseAccessService.getActiveUser(caseId, userAuthorisation);
-        log.info("Logged in user case role type {} on Case ID: {}", loggedInUserCaseRole, caseId);
```

Comment:
Just checking you want this gone, as I don't see it replaced.

Reply:
OK.  I see you've done more than once.

Reply:
get active user - got it.

Reply:
This is duplicate log, the same log is in the assignCaseAccessService.getActiveUser() method call